samples/msal-node-samples/standalone-samples/device-code/package.json

```diff
@@ -12,6 +12,7 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "@azure/msal-node": "file:../../../../lib/msal-node"
+    "@azure/msal-node": "file:../../../../lib/msal-node",
+    "@azure/msal-common": "file:../../../../lib/msal-common"
```

Comment:
This line is not needed as msal-common will be included with msal-node

Reply:
You are right, I added this as changes to the local `msal-common` were not available to the samples.